server/utils/referrals/caseListUtils.ts

```diff
@@ -54,9 +59,57 @@ export default class CaseListUtils {
       queryParams.push({ key: 'status', value: status })
     }
 
+    if (sortColumn && sortDirection) {
+      queryParams.push({ key: 'sortColumn', value: sortColumn })
+      queryParams.push({ key: 'sortDirection', value: sortDirection })
+    }
+
+    return queryParams
+  }
+
+  static queryParamsExcludingSort(audience?: string, status?: string, page?: string): Array<QueryParam> {
+    const queryParams: Array<QueryParam> = []
+
+    if (audience) {
+      queryParams.push({ key: 'strand', value: audience })
+    }
+
+    if (status) {
+      queryParams.push({ key: 'status', value: status })
+    }
+
+    if (page) {
+      queryParams.push({ key: 'page', value: page })
+    }
+
     return queryParams
   }
 
+  static sortableTableHeadings(
+    basePath: string,
+    columns: Record<string, CaseListColumnHeader>,
```

Comment:
Note: This _could_ use the recently added `SortableCaseListColumnKey` but need to make some further changes to how we type incoming the request query params, which is something I think could be done across all the controllers.